dfx.json

```diff
@@ -1,5 +1,25 @@
 {
   "canisters": {
+    "icp_ledger": {
+      "type": "custom",
+      "candid": "candid/ledger.did",
+      "wasm": "./icp_ledger.wasm.gz",
+      "remote": {
+        "id": {
+          "ic": "ryjl3-tyaaa-aaaaa-aaaba-cai"
+        }
+      }
+    },
+    "icp_index": {
+      "type": "custom",
+      "candid": "candid/index.did",
+      "wasm": "./icp_index.wasm.gz",
```

Comment:
update these settings `wasm` is pointing to wrong files

Reply:
Ahh I think the icp_index and ledger looks good. Don't worry about this. My bad.